gradle/extensions/http-servlet/README.markdown

```diff
@@ -0,0 +1,14 @@
+# Custom Servlet on Liferay
+
+This sample demonstrate how to implement HttpServlet on Liferay 7 / DXP. 
+
+## Usage
+1. Set up Liferay Workspace.
+2. Go to ```${liferay_workspace_root}/modules``` and clone this repository
+3. Go to the root folder of this repository that you just cloned
+4. Start up your Liferay bundle
+5. Run ```gradle deploy``` and place the generated jar into ```${liferay_home}/deploy``` or if you have ```blade``` installed, run ```blade deploy```
+6. Sign in to Liferay as Administrator
+7. Navigate to Control Panel -> Configuration -> Server Administration -> Log Levels -> Add Category
+8. Configure ```com.liferay.blade.samples.httpservlet.BladeHttpServlet``` and the log level to ```INFO```
+9. Access to ```http://localhost:8080/o/blade/httpservlet``` You'll see error page (no actual implementations have existed yet) and info log on your console. 
```

Comment:
@yasuflatland-lf  can you explain this error?  What does it mean that `no actual implementations have existed yet`?

Reply:
@gamerson  You can actually access to the page, but it's plain Tomcat's page. What page is it suppose to be displayed? Maybe plain "It worked!" kind of page should be created and displayed?

Reply:
@yasuflatland-lf I followed the instructions on the README, but on step 9, this is what i get in the console,
```18:07:56,848 WARN  [http-nio-8080-exec-8][code_jsp:172] {code="404", msg="ProxyServlet: /blade/httpservlet", uri=/o/blade/httpservlet}```

Then i get redirected to /web/guest/home